### ISSUE TITLE
Support for `NotificationChannel` in Textract Caller's Async Methods

### ISSUE THREAD

**Issue by azucker99** [aws-samples/amazon-textract-textractor#390]
The standard paradigm for using Textract's asynchronous API to process multi-page documents is to [publish the completion status to an SNS topic](https://docs.aws.amazon.com/textract/latest/dg/api-async.html) for consumption by a downstream consumer (e.g., SQS, Lambda, etc.). Currently, the [Textractor method mappings](https://aws-samples.github.io/amazon-textract-textractor/textractor.html#module-textractor.textractor) do not support the `NotificationChannel` parameter required for sending a completion status to an SNS topic:
https://github.com/aws-samples/amazon-textract-textractor/blob/e60857684e9498c24e3802e9b6d323bda10d4ed5/textractor/textractor.py#L425-L439

Adding the option to pass the `notification_channel` to the Textract asynchronous API method mappings would be beneficial in closing the loop on the asynchronous document processing paradigm.

**Comment by Chuukwudi** [comment]
If you need async processing, just use the [existing caller](https://github.com/aws-samples/amazon-textract-textractor/tree/master/caller) or boto3 directly.

This library's real value is in parsing Textract json output, not async handling. What's your specific use case for needing this functionality?

**Comment by azucker99** [comment]
I don't believe I have a unique use case. My argument is that it would be nice for consistency throughout my application using Textractor's abstraction in the calls to Textract as well as parsing the Textract JSON output. Regardless, you're right that it's easy enough to use the existing caller/boto3 so I'm going to close this issue out since the functionality already exists out there in the aforementioned sources.
